### DISCUSSION TITLE
Disable spelling & diagnostics with modeline

### DISCUSSION THREAD

**Discussion by octvs** [LazyVim/LazyVim#1141] [Q&A]
Just starting off, I like the spelling and grammar but for languages which are not supported with languagetool, I need to repeatedly turn off. Current solution is to use default keybindings to turn off both after openning the file, would be nice to have a better solution. Perhaps via modeline, any ideas?

**Comment by octvs** [comment]
Figured the responsible autocmd for it and disabled it :+1:

**Reply by supinum** [reply]
but how?
